source/includes/fundamentals/code-snippets/srv.go

```diff
@@ -14,6 +14,7 @@ const uri = "<connection string>"
 
 func main() {
 
+	// Use the ServerAPI() method to set Stable API version 1
```

Comment:
Suggestion:

I like the brevity of the comment, but think this fragment still needs an article and preposition to identify what's being set and what it's set to.

```suggestion
	// Use the ServerAPI() method to set the Stable API to version 1
```